### PR TITLE
Generate separate phar file for PHP 5.4

### PR DESCRIPTION
**Pull request by Naktibalda**
PHP5.4 phar will be bundled with Guzzle 5.3 and Symfony 2.8
The main phar file will be bundled with the latest version of Guzzle 6 and Symfony 3.

Solves #2124

### CODE REVIEW

File: RoboFile.php [hunk 1]
```diff
@@ -142,13 +145,74 @@ public function testCli()
...
+        $this->installDependencies($dependencies);
+    }
+
+
+
+    private function installDependencies($dependencies)
+    {
+        $composerRequireCommand = str_replace(' install', ' require --no-update ', $this->taskComposerInstall()->getCommand());
+        foreach ($dependencies as $dependency => $version) {
+            $code = $this->taskExec($composerRequireCommand . $dependency . ' ' . $version )->printed(true)->run()->getExitCode();
```
Comment (DavertMik) [inline RoboFile.php]: sometimes Nitpick is just doing too much

**Comment by DavertMik** [comment]
restarted appveyor build, should pass now

File: RoboFile.php [hunk 2]
```diff
@@ -142,13 +145,74 @@ public function testCli()
...
+            'symfony/css-selector' => '2.8.*',
+            'symfony/dom-crawler' => '2.8.*',
+        ];
+
+        $this->installDependencies($dependencies);
+    }
+
+    private function installDependenciesForPhp55()
+    {
+        $dependencies = [
```
Comment (DavertMik) [inline RoboFile.php]: I'd like not to define current dependencies here but use data from `composer.json`.
In current case dependencies are overwritten by those settings from Robofile which is not good and makes releases unpredictable.

I'd recomment to copy current `composer.json`, build php 5.4 version, restore `composer.json` and continue;

**Comment by Naktibalda** [comment]
I found a better way to install version specific dependencies without hardcoding them in Robofile.

**Comment by DavertMik** [comment]
So I think I can merge it?

**Pull request merged**